Modules/ArcGIS/DSCResources/ArcGIS_Server_TLS/ArcGIS_Server_TLS.psm1

```diff
@@ -1022,12 +1022,12 @@ function Get-SSLCertificateForMachine
                 $null 
             }else {
                 $IssuerValue = $null
-                $Issuer = [regex]::matches($res.Content, '"issuer":"([A-Za-z =,\.0-9\-]+)\"')
+                $Issuer = [regex]::matches($res.Content, '"Issuer":[ ]?\"([A-Za-z =,\.0-9\-]+)\"')
```

Comment:
On our site `issuer` is all lowercase. Maybe the check could be case-insensitive?

```[1060-303]:                            [[ArcGIS_Server_TLS]Server_TLS_1060-303.gis.lab] POST http://1060-303.gis.lab:6080/arcgis/admin/machines/1060-303.gis.lab/sslcertificates/1060-303.gis.lab with 101-byte payload
[1060-303]:                            [[ArcGIS_Server_TLS]Server_TLS_1060-303.gis.lab] received 641-byte response of content type application/json;charset=UTF-8
[1060-303]:                            [[ArcGIS_Server_TLS]Server_TLS_1060-303.gis.lab] Response {"aliasName":"1060-303.gis.lab","issuer":"CN=gis-lab-CA, DC=gis, DC=lab","subject":"CN=1060-303.gis.lab","subjectAlternativeNames":["DNSName: 1060-303.gis.lab"],"validFrom":"Fri Feb 23 09:32:19 CET 2018","validUntil":"Sun Feb 23 09:32:19 CET 2020","keyAlgorithm":"RSA","keySize":2048,"serialNumber":"31000000191c9486ab18bfe3ee000000000019","version":3,"signatureAlgorithm":"SHA256withRSA","keyUsage":["Digital_Signature","Key_Encipherment"],"md5Fingerprint":"8f09f3f00086ec447dba928856867583","sha1Fingerprint":"6991c2071dddae2974a97c8eb7e2366e089d3f97","sha256Fingerprint":"962c1194a6c1b73eea26bce6138592a15bb11d629d0af4f1d61e2bbd5325d964"}
[1060-303]:                            [[ArcGIS_Server_TLS]Server_TLS_1060-303.gis.lab] Issuer Value:-
[1060-303]:                            [[ArcGIS_Server_TLS]Server_TLS_1060-303.gis.lab] CN:-